### ISSUE TITLE
fastly_waf_rules output always "latest_revision_number" = 1 even if rules' latest revision is > 1

### ISSUE THREAD

**Issue by dmantellassi** [fastly/terraform-provider-fastly#425]
### Terraform Version
```
❯ terraform -v
Terraform v1.0.0
on darwin_amd64
```

### Affected Resource(s)
- fastly_waf_rules data source

### Terraform Configuration Files
```hcl
terraform {
  required_providers {
    fastly = {
      source = "fastly/fastly"
    }
  }
}

provider "fastly" {
  api_key = "..."
}

data "fastly_waf_rules" "test" {
  tags = [
    "capec/1000/225/21/593/61"   // or any filter that extract rules with latest revision > 1
  ]
}

output "fastly_waf_data" {
  value = data.fastly_waf_rules.test
}
```

### Debug Output
https://gist.github.com/dmantellassi/7713b612a2e5797b85150f23d09b8d28

### Expected Behavior
Rules that have more than one revision should report the correct latest_revision_number in fastly_waf_rules data source.

### Actual Behavior
fastly_waf_rules data source always output `"latest_revision_number" = 1` even if the real latest revision is greater, for instance take rule "modsec_rule_id": 943110 that is included in the output of the above TF config.

### Steps to Reproduce
`terraform apply`

**Comment by marco-commercelayer** [comment]
this is fixed by just adding this line in **fastly/data_source_waf_rules.go**
```
@@ -68,32 +75,47 @@ func dataSourceFastlyWAFRulesRead(_ context.Context, d *schema.ResourceData, met
 
 	conn := meta.(*FastlyClient).conn
 	input := &gofastly.ListAllWAFRulesInput{}
+	input.Include = "waf_rule_revisions"
```

**Comment by bengesoff** [comment]
@marco-commercelayer I've thrown it into a PR, thanks for suggesting this. @dmantellassi I'm pretty sure this will fix what you're seeing but please feed back if it doesn't, once it gets merged/released 👍🏼

**Comment by Integralist** [comment]
This fix is in v0.33.0 https://github.com/fastly/terraform-provider-fastly/releases/tag/v0.33.0
